x/auth/vesting/types/codec.go

```diff
@@ -53,6 +53,7 @@ func RegisterInterfaces(registry types.InterfaceRegistry) {
 	registry.RegisterImplementations(
 		(*sdk.Msg)(nil),
 		&MsgCreateVestingAccount{},
+		&MsgCreatePermanentLockedAccount{},
```

Comment:
I didn't end up having to add `MsgCreatePermanentLockedAccount` to the `RegisterLegacyAminoCodec` block up above (line 14). I'm not totally sure why... but I managed to test on my Ledger device with simapp, and I successfully used the CLI for creating & signing new permanent locked accounts.